pkg/controllermanager/apis/config/v1alpha1/types.go

```diff
@@ -61,6 +61,8 @@ type ControllerManagerControllerConfiguration struct {
 	// Event defines the configuration of the Event controller.  If unset, the event controller will be disabled.
 	// +optional
 	Event *EventControllerConfiguration `json:"event,omitempty"`
+	// ExposureClass defines the configuration of the ExposureClass controller.
+	ExposureClass *ExposureClassControllerConfiguration `json:"exposureClass,omitempty"`
```

Comment:
Please enhance https://github.com/gardener/gardener/blob/master/example/20-componentconfig-gardener-controller-manager.yaml and https://github.com/gardener/gardener/blob/master/charts/gardener/controlplane/charts/runtime/templates/controller-manager/configmap-componentconfig.yaml

Reply:
The chart is updated. As said I will add examples in a subsequent pr.